Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -909,28 +920,49 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
             notifyCursorChanged(toWebCoreCursorType(pdfElementTypes, altKeyIsActive));
             return true;
         }
+        case WebMouseEventButton::Left: {
+            if (m_trackedAnnotation && m_trackedAnnotation != annotationForRootViewPoint(event.position())) {
+                handleMouseDraggedOffTrackedAnnotation();
+                return true;
+            }
+            return false;
+        }
         default:
             return false;
         }
     case WebEventType::MouseDown:
         switch (event.button()) {
         case WebMouseEventButton::Left: {
-            auto pointInDocumentSpace = convertFromPluginToDocument(convertFromRootViewToPlugin(event.position()));
-            auto nearestPageIndex = nearestPageIndexForDocumentPoint(pointInDocumentSpace);
-            if (!nearestPageIndex)
+            if (RetainPtr<PDFAnnotation> annotation = annotationForRootViewPoint(event.position()).value_or(nullptr)) {
```

Comment:
If we didn't return `std::optional<...>` here we could just do this here
```suggestion
            if (RetainPtr<PDFAnnotation> annotation = annotationForRootViewPoint(event.position())) {
```

Reply:
Done!